web_ir_actions_act_window_message/static/src/js/web_ir_actions_act_window_message.esm.js

```diff
@@ -0,0 +1,19 @@
+/** @odoo-module **/
+
+import {ActWindowMessageDialog} from "./web_ir_actions_act_window_msg_component.esm";
+import {registry} from "@web/core/registry";
+
+// Define a function to open the dialog
+function openDialog({env, action}) {
+    // Created new Dialog widget.
+    env.services.dialog.add(ActWindowMessageDialog, {
+        title: action.title,
+        body: action.message,
```

Comment:
```suggestion
        action.is_html_message ? markup(action.message) : action.message,
```

you need to explicitly mark html as safe by now to avoid escaping